lib/features/wallet_screen/widget/transaction_item.dart

```diff
@@ -0,0 +1,84 @@
+part of '../../feature.dart';
+
+class TransactionItem extends StatelessWidget {
+  final String description;
+  final String date;
+  final String amount;
+  final Color? iconBgColor;
+  final bool isWithdrawal;
+  final bool showGreenAmount;
+
+  const TransactionItem({
+    Key? key,
+    required this.description,
+    required this.date,
+    required this.amount,
+    this.iconBgColor,
+    required this.isWithdrawal,
+    this.showGreenAmount = false,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      padding: const EdgeInsets.all(16),
+      decoration: BoxDecoration(
+        color: Colors.white,
```

Comment:
don't use colors like this see how colors are referenced from other screens/widget